nengo/spa/tests/test_assoc_mem.py

```diff
@@ -35,3 +37,60 @@ def input_func(t):
     # Check to see if model builds properly. No functionality test needed
     with Simulator(m):
         pass
+
+
+def test_am_spa_keys_as_expressions(Simulator, plt, seed, rng):
+    """Provide semantic pointer expressions as input and output keys."""
+    D = 64
+
+    vocab_in = Vocabulary(D, rng=rng)
+    vocab_out = Vocabulary(D, rng=rng)
+
+    vocab_in.parse('A+B')
+    vocab_out.parse('C+D')
+
+    in_keys = ['A', 'A*B']
+    out_keys = ['C*D', 'C+D']
+
+    with nengo.spa.SPA(seed=seed) as model:
+        model.am = AssociativeMemory(input_vocab=vocab_in,
+                                     output_vocab=vocab_out,
+                                     input_keys=in_keys,
+                                     output_keys=out_keys)
+
+        model.inp = Input(am=lambda t: 'A' if t < 0.1 else 'A*B')
+
+        in_p = nengo.Probe(model.am.input)
+        out_p = nengo.Probe(model.am.output, synapse=0.03)
+
+    with nengo.Simulator(model) as sim:
+        sim.run(0.2)
+
+    # Specify t ranges
+    t = sim.trange()
+    t_item1 = (t > 0.075) & (t < 0.1)
```

Comment:
I didn't know that you can use the `&` operator; I always used `np.logical_and`. This is so much more convenient! :)